app/javascript/custom-tagged-plate/components/CustomTaggedPlate.vue

```diff
@@ -597,10 +598,10 @@ export default {
         })
         .catch((error) => {
           // Something has gone wrong
-          this.loading = false
           // TODO Replace this with generic limber logging when available
```

Comment:
Can probably remove this comment?

Reply:
Sure makes sense. I may be wrong but I don't think think its possible to log to system level with these js files since they are client-side so not sure how an integration like that would work.